### PR TITLE
Update safety-scanner-download.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document to reduce ambiguity

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9533

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/intelligence/safety-scanner-download.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Microsoft Safety Scanner is a scan tool designed to find and remove malware from
 
 - The security intelligence update version of the Microsoft Safety Scanner matches the version described [in this web page](https://www.microsoft.com/wdsi/definitions).
 
-- Safety Scanner only scans when manually triggered and is available for use 10 days after being downloaded. We recommend that you always download the latest version of this tool before each scan.
+- Safety Scanner only scans when manually triggered. The Microsoft Safety Scanner expires 10 days after being downloaded. To rerun a scan with the latest anti-malware definitions, download and run the Microsoft Safety Scanner again. We recommend that you always download the latest version of this tool before each scan.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/intelligence/safety-scanner-download.md]:
```suggestion
- Microsoft Safety Scanner only scans when manually triggered. Safety Scanner expires 10 days after being downloaded. To rerun a scan with the latest anti-malware definitions, download and run Safety Scanner again. We recommend that you always download the latest version of this tool before each scan.
```

File: microsoft-365/security/intelligence/safety-scanner-download.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Microsoft Safety Scanner is a scan tool designed to find and remove malware from
...
 
 - Safety scanner is a portable executable and does not appear in the Windows Start menu or as an icon on the desktop. Note where you saved this download.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/intelligence/safety-scanner-download.md]:
```suggestion
- Safety Scanner is a portable executable and does not appear in the Windows Start menu or as an icon on the desktop. Note where you saved this download.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@msbemba Clear validation check please. Thanks

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@Dansimp Gentle reminder.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by msbemba** [comment]
It has succeeded now @yogkumgit

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**